backend/v1/project.py

```diff
@@ -130,6 +130,7 @@ def create_project(current_user):
 
 @projects_api.route('/<int:project_id>', methods=['DELETE'])
 @token_required
+@admin_required
```

Comment:
Nu mag alleen een admin projects verwijderen. Wat als een oprichter van het project dat wil doen? Of moet die eerst een aanvraag doen?